recipe/meta.yaml

```diff
@@ -1,50 +1,102 @@
 {% set name = "polars" %}
-{% set version = "0.18.3" %}
+{% set version = "0.20.18" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/polars-{{ version }}.tar.gz
-    sha256: efc3f629fddb060dc90c2dbd575824c1d662f37294c18c769d28e2bc4d5a1413
+    sha256: 8a321cbdbb459e3c0cc1af2ce6ac930d0d3b5ccbeb2dd3e4237ad07d487fd290
 
 build:
   number: 0
-  skip: true  # [py<37]
-  skip: true  # [win and python_impl=="pypy"]
+  # no rust nightly compiler for s390x
+  skip: true  # [s390x]
+  # skipping windows because missing maturin >=1.3.2 and since win is not a priority for SF
+  skip: true  # [win]
 
 requirements:
   build:
-    - python                              # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
-    - crossenv                            # [build_platform != target_platform]
-    - maturin >=0.14,<0.15                # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('rust') }}
-    - posix                               # [win]
+    - python
+    - posix   # [win]
     - cmake
-    - make                                # [unix]
+    - make    # [unix]
     - cargo-bundle-licenses
+    - pip     # [linux and x86_64]
```

Comment:
I got errors on `win64` for a few rust packages if `python`/`maturin` are missing in `build`. See https://github.com/AnacondaRecipes/watchfiles-feedstock/commit/4aeeb2aeb9ca6841556b56693d886b6ce8fadd33.
But we [updated](https://github.com/AnacondaRecipes/aggregate/blob/4b7daa553fb97068e294f13b2ab048e645d9b920/conda_build_config.yaml#L36-L44) `conda_build_config.yaml` and now it might be not affecting. I'm not sure 100%.

Reply:
yeah this is weird indeed. I think I got the same error but I also changed the build script so I don't call pip explicitly anymore. I will see if this solves the issue.